department-of-veterans-affairs/vets-api
# API-38465-claim-status-404-error-format

## Summary

-  Adds a way to handle errors.messages
- Fixes a minor bug in pdf_client.


## Related issue(s)

- [API-38465](https://jira.devops.va.gov/browse/API-38465)

## Testing done

- Postman
1. run {{uri}}/services/claims/v2/veterans/1012666073V986297/526/synchronous, using the following request_body:
```               
             "mailingAddress": {
                    "addressLine1": "1234 Couch Street",
                    "addressLine2": "Apt 4",
                    "city": "Portland",
                    "state": "OR",
                    "country": "France",
                    "internationalPostalCode": "105-011"
                },
```
2. grab the claim_id from logs
3. call {{uri}}/services/claims/v1/claims/{{claim_id from first call}}
4. Verify that the 2nd call does not return a 404


## What areas of the site does it impact?
- claims controller v1
- pdf_client

## Acceptance criteria
